LunaBot/Commands/User/GetAttributeCommand.cs

```diff
@@ -377,4 +377,39 @@ class GetPrivilegeCommand : BaseCommand
             }
         }
     }
+
+    [LunaBotCommand("get_snug")]
```

Comment:
if you do go with `s` as a short for this command then you should also add "get_s" here as well